### PR TITLE
add more step-by-step instructions to the readme

### PR DESCRIPTION
**Pull request by keachi**
I added a bash script to build the environment of a new training from skeleton. I know only a few steps are needed, but it was requested as a feature.

### CODE REVIEW

**Comment by eni23** [comment]
Nice. Just one thing: we should not have Makefiles inside the skeleton, since this will not do the same as the training-builder from commons. Also consider to move the skeleton-script to commons, since there should be only content and no code in the main repo.

File: README.md [hunk 1]
```diff
@@ -3,16 +3,48 @@
...
+
+- Edit `<training-name>/<id>_<module-name>/modul_<module-name>_<id>.yml` to
+  your needs. This file is required to build the training.
 
-- Edit `training-name/module-name/*yml` to your needs. This file is required to build the training.
 - Add your training content
-  - All images needs to be in a subdirectory of `training-name/module-name/`,
-  eg.  `training-name/module-name/static`. Otherwise they will not display when builded
+
+  - All images needs to be in a subdirectory of
```
Comment (karras) [inline README.md]: s/needs/need/

File: README.md [hunk 2]
```diff
@@ -3,16 +3,48 @@
...
+    `<training-name>/<id>_<module-name>/`,
+    e.g. `<training-name>/<id>_<module-name>/`, otherwise they will not display
+    when builded
```
Comment (karras) [inline README.md]: s/builded/built/

File: README.md [hunk 3]
```diff
@@ -3,16 +3,45 @@
...
-- Edit `training-name/module-name/*yml` to your needs. This file is required to build the training.
 - Add your training content
-  - All images needs to be in a subdirectory of `training-name/module-name/`,
-  eg.  `training-name/module-name/static`. Otherwise they will not display when builded
-  - Make sure to only use non-copyrighted materials
 
-- Add the files to the git repository and push it. After a few minutes, the new training should be available at https://docs.adfinis-sygroup.ch/public/trainings
+    - All images need to be in a subdirectory of
+      `<training-name>/<id>_<module-name>/`,
+      e.g. `<training-name>/<id>_<module-name>/`, otherwise they will not display
```
Comment (karras) [inline README.md]: not be displayed...

**Pull request merged**